packaging/bundle-validation/ci_run.sh

```diff
@@ -96,14 +99,73 @@ fi
+    HUDI_UTILITIES_SLIM_BUNDLE_NAME=hudi-utilities-slim-bundle_2.11
+  elif [[ ${SPARK_PROFILE} == 'spark2.4' ]]; then
+    HUDI_SPARK_BUNDLE_NAME=hudi-spark2.4-bundle_2.11
+    HUDI_UTILITIES_BUNDLE_NAME=hudi-utilities-bundle_2.11
+    HUDI_UTILITIES_SLIM_BUNDLE_NAME=hudi-utilities-slim-bundle_2.11
+  elif [[ ${SPARK_PROFILE} == 'spark3.1' ]]; then
+    HUDI_SPARK_BUNDLE_NAME=hudi-spark3.1-bundle_2.12
+    HUDI_UTILITIES_BUNDLE_NAME=hudi-utilities-bundle_2.12
+    HUDI_UTILITIES_SLIM_BUNDLE_NAME=hudi-utilities-slim-bundle_2.12
+  elif [[ ${SPARK_PROFILE} == 'spark3.2' ]]; then
+    HUDI_SPARK_BUNDLE_NAME=hudi-spark3.2-bundle_2.12
+    HUDI_UTILITIES_BUNDLE_NAME=hudi-utilities-bundle_2.12
+    HUDI_UTILITIES_SLIM_BUNDLE_NAME=hudi-utilities-slim-bundle_2.12
+  elif [[ ${SPARK_PROFILE} == 'spark3.3' ]]; then
+    HUDI_SPARK_BUNDLE_NAME=hudi-spark3.3-bundle_2.12
+    HUDI_UTILITIES_BUNDLE_NAME=hudi-utilities-bundle_2.12
+    HUDI_UTILITIES_SLIM_BUNDLE_NAME=hudi-utilities-slim-bundle_2.12
+  elif [[ ${SPARK_PROFILE} == 'spark3' ]]; then
+    HUDI_SPARK_BUNDLE_NAME=hudi-spark3-bundle_2.12
+    HUDI_UTILITIES_BUNDLE_NAME=hudi-utilities-bundle_2.12
+    HUDI_UTILITIES_SLIM_BUNDLE_NAME=hudi-utilities-slim-bundle_2.12
+  fi
+
+  if [[ ${FLINK_PROFILE} == 'flink1.13' ]]; then
+    HUDI_FLINK_BUNDLE_NAME=hudi-flink1.13-bundle
+  elif [[ ${FLINK_PROFILE} == 'flink1.14' ]]; then
+    HUDI_FLINK_BUNDLE_NAME=hudi-flink1.14-bundle
+  elif [[ ${FLINK_PROFILE} == 'flink1.15' ]]; then
+    HUDI_FLINK_BUNDLE_NAME=hudi-flink1.15-bundle
+  elif [[ ${FLINK_PROFILE} == 'flink1.16' ]]; then
+    HUDI_FLINK_BUNDLE_NAME=hudi-flink1.16-bundle
+  elif [[ ${FLINK_PROFILE} == 'flink1.17' ]]; then
+    HUDI_FLINK_BUNDLE_NAME=hudi-flink1.17-bundle
+  fi
+
+  echo "Downloading bundle jars from staging repo orgapachehudi-$STAGING_REPO_NUM ..."
+  REPO_BASE_URL=https://repository.apache.org/content/repositories/orgapachehudi-$STAGING_REPO_NUM/org/apache/hudi
+  wget -q $REPO_BASE_URL/$HUDI_FLINK_BUNDLE_NAME/$HUDI_VERSION/$HUDI_FLINK_BUNDLE_NAME-$HUDI_VERSION.jar -P $TMP_JARS_DIR/
+  wget -q $REPO_BASE_URL/$HUDI_HADOOP_MR_BUNDLE_NAME/$HUDI_VERSION/$HUDI_HADOOP_MR_BUNDLE_NAME-$HUDI_VERSION.jar -P $TMP_JARS_DIR/
+  wget -q $REPO_BASE_URL/$HUDI_KAFKA_CONNECT_BUNDLE_NAME/$HUDI_VERSION/$HUDI_KAFKA_CONNECT_BUNDLE_NAME-$HUDI_VERSION.jar -P $TMP_JARS_DIR/
+  wget -q $REPO_BASE_URL/$HUDI_SPARK_BUNDLE_NAME/$HUDI_VERSION/$HUDI_SPARK_BUNDLE_NAME-$HUDI_VERSION.jar -P $TMP_JARS_DIR/
+  wget -q $REPO_BASE_URL/$HUDI_UTILITIES_BUNDLE_NAME/$HUDI_VERSION/$HUDI_UTILITIES_BUNDLE_NAME-$HUDI_VERSION.jar -P $TMP_JARS_DIR/
+  wget -q $REPO_BASE_URL/$HUDI_UTILITIES_SLIM_BUNDLE_NAME/$HUDI_VERSION/$HUDI_UTILITIES_SLIM_BUNDLE_NAME-$HUDI_VERSION.jar -P $TMP_JARS_DIR/
+  wget -q $REPO_BASE_URL/$HUDI_METASERVER_SERVER_BUNDLE_NAME/$HUDI_VERSION/$HUDI_METASERVER_SERVER_BUNDLE_NAME-$HUDI_VERSION.jar -P $TMP_JARS_DIR/
+  echo "Downloaded these jars from $REPO_BASE_URL for validation:"
```

Comment:
do you think we should add a count validation on the output of 
```
ls -l $TMP_JARS_DIR
```

Reply:
I think it's not needed.  If a jar is missing, the validation logic will fail.